package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1158,11 +1160,13 @@ def _write_timestep(self, ts, multiframe=False):
         if multiframe:
             self.MODEL(self.frames_written + 1)
 
+        prev_atom = {}
+        serial_count = 0
         for i, atom in enumerate(atoms):
             segid = atom.segid if atom.segid is not "SYSTEM" else " "
 
             vals = {}
-            vals['serial'] = int(str(i + 1)[-5:])  # check for overflow here?
+            vals['serial'] = serial_count + 1
```

Comment:
Oh, Thanks for the feedback. I was actually wondering why do we had to do all that for just incrementing the counter. Now I understood that and will make changes accordingly.